spec/requests/api/visitor_can_view_index_list_of_foodbags_spec.rb

```diff
@@ -0,0 +1,17 @@
+RSpec.describe "GET /api/foodbags" do
+  let!(:foodbags) {5.times{create(:foodbag)}}
```

Comment:
maybe a test for list of several foodbags..

Reply:
Why do you think that? My thoughts is that is if its fly once, it should fly 5 times.